README.md

```diff
@@ -1,2 +1,166 @@
-# SciPy2020-Proposal
-SciPy 2020 proposal for a poster on pyhf
+# [SciPy 2020](https://www.scipy2020.scipy.org/) Submission Instructions
+
+## Author 1
+
+- First Name: Matthew
+- Last Name: Feickert
+- Email: matthew.feickert at-no-spam cern.ch
+- Country: United States
+- Organization: University of Illinois at Urbana-Champaign
+- Web page: http://www.matthewfeickert.com/
+- Corresponding Author: Yes
+
+## Author 2
+
+- First Name: Lukas
+- Last Name: Heinrich
+- Email: lukas.heinrich at-no-spam cern.ch
+- Country: Germany
+- Organization: CERN
+- Web page: http://www.lukasheinrich.com/
+- Corresponding Author: No
+
+## Author 3
+
+- First Name: Giordon
+- Last Name: Stark
+- Email: gstark at-no-spam cern.ch
+- Country: United States
+- Organization: University of California Santa Cruz Santa Cruz Institute for Particle Physics (UCSC SCIPP)
+- Web page: https://giordonstark.com/
+- Corresponding Author: No
+
+## Title and Abstract
+> The title and the abstract should be entered as plain text, they should not contain HTML elements.
+
+### Title
+
+pyhf: a pure Python statistical fitting library with tensors and autograd
```

Comment:
```suggestion
pyhf: a Python-only statistical fitting library with tensors and autograd
```

Reply:
Why the change here? We've used "pure Python" to describe it in all work in the past.